static/css/style.css

```diff
@@ -885,6 +912,11 @@ input:checked + .menu-switch-slider:before {
     background-color: #eee;
     color: black;
 }
+
+.save-button.py:focus {
+  box-shadow: 0px 0px 5px 3px #FFCC33;
+}
+
```

Comment:
The number of spaces for the CSS in this file is a bit of a mess, but I think right now there is more classes using 4 spaces than 2. Could we update all new classes use to use 4 spaces as well?

Reply:
Sorry, I meant the 4 spaces like this:

```css
.save-button.py:focus {
  box-shadow: 0px 0px 5px 3px #FFCC33;
}
```

To:

```css
.save-button.py:focus {
    box-shadow: 0px 0px 5px 3px #FFCC33;
}
```